main.go

```diff
@@ -68,23 +69,16 @@ func RealMain() {
 					resp.Write([]byte("OK"))
 					return
 				}
+
 				if utils.IsGrpcRequest(req) {
 					grpcapi.GrpcHandlerFunc(resp, req)
-				} else if req.URL.Path == "/verify" {
-					if err := httpapi.HandleVerification(req); err != nil {
-						respText := fmt.Sprintf("%s", err.Error())
-						resp.Write([]byte(respText))
-					} else {
-						resp.Write([]byte("Successfully verified account!"))
-					}
 				} else {
-					resp.WriteHeader(http.StatusBadRequest)
-					resp.Write([]byte("Invalid URL requested"))
+					httpapi.HttpMux.ServeHTTP(resp, req)
 				}
 			},
 		),
 	}
-	models.InspectComponents()
```

Comment:
i removed this @abesheknarayan , your thoughts ?

Reply:
we can remove it for now and add it later once we have a modified cheat suite